hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMOpenKeysDeleteRequest.java

```diff
@@ -88,25 +87,19 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     OMClientResponse omClientResponse = null;
     Result result = null;
     Map<String, OmKeyInfo> deletedOpenKeys = new HashMap<>();
-    Map<String, OmBucketInfo> updatedBuckets = new HashMap<>();
 
     try {
       for (OpenKeyBucket openKeyBucket: submittedOpenKeyBuckets) {
         // For each bucket where keys will be deleted from,
         // get its bucket lock and update the cache accordingly.
-        OmBucketInfo omBucketInfo = updateTableCachePerBucket(
-            ozoneManager, trxnLogIndex, openKeyBucket, deletedOpenKeys);
```

Comment:
Why not continue to pass `deletedOpenKeys` and avoid the additional allocations?